lib/logic/player/favorites.dart

```diff
@@ -41,13 +39,15 @@ class FavoritesControl with Control {
 
   @override
   void dispose() {
-    _favoriteSetsMap.clear();
+    for (var element in _favoriteSetsMap.entries) {
+      element.value.clear();
+    }
```

Comment:
refactor this back, when you add ContentMapClearExtension

Reply:
Are you sure setting them to null is a good idea? This means I have to make the type nullable and nullcheck on access.

Reply:
Ok, let's keep it like this